### ISSUE TITLE
Is the modelPath algorithm missing from the code? Unable to find algorithm content！

### ISSUE THREAD

**Issue by 1512033796** [dnutiu/image-tagger#1]
Is the modelPath algorithm missing from the code? Unable to find algorithm content！

**Comment by dnutiu** [comment]
Hi,

I'm not exactly sure what you're referring to.

The model inference algorithm can be found here: https://github.com/dnutiu/image-tagger/blob/main/image-tagger-core/src/main/java/dev/nuculabs/keyworder/ModelInference.java

Let me know if you need help with something else :D

**Comment by 1512033796** [comment]
> Let me know if you need help with something else :D

Hi,D, I ran the main program in ImageTaggerGUI and after selecting an image, an error message was reported. The error message is as follows. I found that the modelPath parameter path in the ChoseImageListener class is empty. How can I solve this problem?

![image](https://github.com/dnutiu/image-tagger/assets/45383053/e6a89e35-58ec-4139-a72e-4125c68885d3)
